Improve updating withdrawal credentials docs (#448)
ConsenSys/doc.teku: docs/HowTo/Withdrawal-Keys.md

withdrawal key.

Tools such as

> [`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli#generate-bls-to-execution-change-arguments) or [`ethdo`](https://github.com/wealdtech/ethdo/blob/master/docs/changingwithdrawalcredentials.md) can sign the request correctly. The signed withdrawal credential change can then be submitted to your own beacon node via the [`bls_to_execution_changes`](https://consensys.github.io/teku/#tag/Beacon/operation/postBlsToExecutionChange) API or it could be broadcasted via [`beaconcha.in`](https://beaconcha.in/tools/broadcast)

Reviewer: ```suggestion
[`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli#generate-bls-to-execution-change-arguments) or [`ethdo`](https://github.com/wealdtech/ethdo/blob/master/docs/changingwithdrawalcredentials.md) can sign the request correctly. You can then submit the signed withdrawal credential change to your own beacon node using the [`bls_to_execution_changes`](https://consensys.github.io/teku/#tag/Beacon/operation/postBlsToExecutionChange) API, or broadcast it using [`beaconcha.in`](https://beaconcha.in/tools/broadcast).
```

Author: Thanks, I like the changes! 👍
